installer/mac/updates/updates.xml

```diff
@@ -6,22 +6,23 @@
     <description>Most recent changes with links to updates.</description>
     <language>en</language>
       <item>
-        <title>Chain Core Developer Edition 1.1.0</title>
+        <title>Chain Core Developer Edition 1.1.1</title>
         <description>
           <![CDATA[
-            <p><b>NOTE:</b> This update will create a fresh blockchain; please backup
-              any blockchain data you wish to preserve.
+            <p><b>NOTE:</b> This update will create a fresh blockchain if you were previously
+              using versions 1.0.x; please backup any blockchain data you wish to preserve.
```

Comment:
I have typically stacked app update release notes with the following format and it works really well:

```
Hey check out these changes:

- Some bug fix

In the previous 1.1 release:

- Keep these release notes going forward until 1.2
- And these too
```